File: java/program67.java
import java.util.*;

class ArrayX
{
    public int Arr[];

    ArrayX(int iSize)
    {
        Arr = new int[iSize];
    }

    public void Accept()
    {
        System.out.println("Enter element of array : ");
        Scanner sobj = new Scanner(System.in);

        for(int i = 0; i < Arr.length; i++)
        {
            Arr[i] = sobj.nextInt();
        }
    }

    public void Display()
    {

        System.out.println("element of array are : ");

        for(int i = 0; i < Arr.length; i++)
        {
            System.out.print(Arr[i]+"\t");
        }
        System.out.println();
    }

    public void MaximumMinimum()
    {
        int iCnt = 0;
        int Max = Arr[iCnt];
        int Min = Arr[iCnt];

        for(iCnt = 0; iCnt < Arr.length; iCnt++)
        {
            if(Arr[iCnt] < Min)
            {
                Min = Arr[iCnt];
            }
            else if(Arr[iCnt] > Max)
            {
                Max = Arr[iCnt];
            }
        }
        System.out.println("Maximum element is : "+Max);
        System.out.println("Minimum element is : "+Min);
    }
}

class program67
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("enter number of elements");
        int size = sobj.nextInt();

        ArrayX obj = new ArrayX(size);
        obj.Accept();
        obj.Display();
        obj.MaximumMinimum();
    }
}

File: cpp/program67.cpp
// program86.c

#include<iostream>
using namespace std;

class Array
{
    public:
        int iSize;
        int *Arr;

        Array(int);
        ~Array();

        void Accept();
        void Display();

        void MaximumMinimum();
};

Array::Array(int X)
{
    iSize = X;
    Arr = new int[iSize];
}

Array::~Array()
{
    delete Arr;
}

void Array::Accept()
{
    cout<<"Enter elements of array \n";

    for(int iCnt = 0; iCnt < iSize; iCnt++)
    {
        cin>>Arr[iCnt];
    }
}

void Array::Display()
{
    cout<<"Elements of the array \n";

    for(int iCnt = 0; iCnt < iSize; iCnt++)
    {
        cout<<Arr[iCnt]<<"\t";
    }
    cout<<"\n";
}

void Array::MaximumMinimum()
{
    int iCnt = 0;
    int iMax = Arr[iCnt];
    int iMin = Arr[iCnt];

    for(int iCnt = 0; iCnt < iSize; iCnt++)
    {
        if(iMax < Arr[iCnt])
        {
            iMax = Arr[iCnt];
        }
        else if(iMin > Arr[iCnt])
        {
            iMin = Arr[iCnt];
        }
    }
    cout<<"Largest element of the array is : "<<iMax<<"\n";
    cout<<"Smallest element of the array is : "<<iMin<<"\n";
}

int main()
{
    int iSize = 0;
    
    cout<<"Enter number of elements \n";
    cin>>iSize;

    Array obj(iSize);
    obj.Accept();
    obj.Display();
    obj.MaximumMinimum();

    return 0;
}